### DISCUSSION TITLE
[Projects Beta] filter iteration field by 'current'

### DISCUSSION THREAD

**Discussion by yinonov** [community/community#8128] [Projects and Issues] [answered]
setting up a `current` sprint view is not available as filter isn't aware of `current` tag 
![image](https://user-images.githubusercontent.com/15140652/143542743-c5258853-ede2-4b29-aa0e-483d8e42a63b.png)

the filter can have a special reference to the `current` tag of an `iteration` field
![image](https://user-images.githubusercontent.com/15140652/143542515-24cbb9f8-5207-4a58-b087-b69f313b3a0e.png)

This will move the projects forward to enable one of the common existing ecosystem workflows

**Comment by joshuaNathaniel** [comment]
This is a no-brainer here. I'm currently running an action at the end of our sprints to just update a label `current` to the current sprint issues then filter by that label. Good enough workaround for something that should be easy.

**Reply by yinonov** [reply]
GH action running from 1 repo and affects all other repos integrated in the project?
Or just external corn job?

**Comment by scottrobertson** [comment]
Yes please! Right now we will just update the filter every iteration, but it would be super nice to just have this set to "Current".

**Comment by zaporylie** [comment, marked as answer]
It is now possible to filter by current sprint by using the `"@current"` value for the `iteration` field, ex. `sprint:"@current"`

**Reply by yinonov** [reply]
Thanks @zaporylie

**Reply by scottrobertson** [reply]
> Thanks @zaporylie 

Thanks for commenting! @zaporylie originally said "not" so I just deleted the email without checking here 🤣

Thanks GitHub team!

**Reply by zaporylie** [reply]
Yes, sorry for the confusion @scottrobertson. Only realized about the mistake after the comment from @yinonov 😅

**Reply by scottrobertson** [reply]
@zaporylie it's all good 😄

**Reply by Alex-ley-scrub** [reply]
there is also support for:
`sprint:@previous,@current,@next`
which is awesome

**Comment by naquiroz** [comment]
Is there way to filter by the next iteration? Something like `sprint:"@current + 1"`?

**Reply by yinonov** [reply]
> Is there way to filter by the next iteration? Something like `sprint:"@current + 1"`?
> 
> 

Or even all _less /greater than_

**Reply by Alex-ley-scrub** [reply]
there is now also support for:
`sprint:@current,@next,@previous`

and 
`-sprint:<@previous -sprint:>@next `

which are awesome

**Comment by arrowcircle** [comment]
Is it possible to filter to include only the current sprint or not set sprint?

**Reply by JonasWanke** [reply]
Only the current sprint: `sprint:@current`
Issues without a sprint: `no:sprint`

(Where `sprint` is the column name)

**Reply by arrowcircle** [reply]
And how to filter current OR no sprint?

**Reply by JonasWanke** [reply]
You can use `-sprint:<@current -sprint:>@current`. There's no specific “or” operator to my knowledge, but a similar feature was requested for labels: https://github.com/orgs/community/discussions/4507

**Reply by Alex-ley-scrub** [reply]
there is also support for:
`sprint:@current,@next,@previous`

and 
`-sprint:<@previous -sprint:>@next `

which are awesome
